app/controllers/application_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  add_flash_types :err_login, :succes_singup
+  include SessionsHelper
```

Comment:
```suggestion
  include SessionsHelper
  
  add_flash_types :err_login, :succes_singup
  
```
Giữa các line của các loại khác nhau nên có 1 line trống để clear code.

include lên đầu e nha